### PR TITLE
Use oldest-supported-numpy in wheel building

### LINKED ISSUE #129: oldest-supported-numpy needs to go into wheel building
In #127 I removed the reference to ``oldest-supported-numpy`` from setup.py as I thought that this was an extra check not related to the wheel building process. But I think this is actually how the wheel building process was fixing the numpy version. (I thought from earlier incarnations there was a requirements.txt for this).

We should be able to install ``oldest-supported-numpy`` at the same time as we ``pip install setuptools wheel``, right?

### PR DESCRIPTION
**Pull request by benjeffery**
Fixes #129

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -24,10 +24,13 @@ jobs:
         env:
           PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
         shell: bash
+        # Instead of letting setup.py install a newer numpy we install it here
+        # using the oldest supported version for ABI compatibility
         run: |
           set -ex
           ${PYTHON} -m pip install --upgrade pip
           ${PYTHON} -m pip install setuptools wheel cython
```
Comment (jeromekelleher) [inline .github/workflows/wheels.yml:32]: Does cython pull in numpy? If so, maybe better to install oldest-supported-numpy first?

Comment (benjeffery) [reply .github/workflows/wheels.yml]: Removed cython

**Comment by benjeffery** [comment]
@jeromekelleher Removed cython - should be ready to go.

**Comment by jeromekelleher** [comment]
Great, thanks @benjeffery. Let's give it a go!

**Pull request merged**